composer.json

```diff
@@ -18,9 +18,9 @@
     "require": {
         "php": "^7.2",
         "jms/metadata": "~1.1",
-        "jms/parser-lib": "1.*",
         "doctrine/annotations": "^1.0",
-        "doctrine/instantiator": "^1.0.3"
+        "doctrine/instantiator": "^1.0.3",
+        "hoa/compiler": "^3.17.08.08"
```

Comment:
Out of curiosity, which versioning scheme does HOA follows? Is this some kind of 3.x?

Reply:
If I understand correctly, it's <major>.<releaseyear>.<releasemonth>.<releaseday>. Looks pretty weird, but incidentally works with semver. :joy: